src/services/services.ts

```diff
@@ -1807,6 +1807,15 @@ namespace ts {
             return range && createTextSpanFromRange(range);
         }
 
+        function getSpanForPosition(fileName: string, position: number): TextSpan {
+            synchronizeHostData();
+
+            const sourceFile = getValidSourceFile(fileName);
+            const node = getTouchingPropertyName(sourceFile, position, /*includeJsDocCcomment*/ false);
+

```

Comment:
also `SpanForPosition` seems a bit vauge to me.. is this `TokenSpanAtPosition` or `NodeSpanAtPosition` or `IdentiiferTokenSpanAtPosition`?

Reply:
No longer valid as the method has been removed.